test/x509_time_test.c

```diff
@@ -25,6 +25,46 @@ typedef struct {
     int expected;
 } TESTDATA;
 
+/*
+ * Actually, the "loose" mode has been tested in
+ * those time-compare-cases, so we may not test it again.
+ */
+static TESTDATA x509_strict_tests[] = {
+    {
+        "20170217180154Z", V_ASN1_GENERALIZEDTIME,
+        0, 1,
```

Comment:
It might also be useful to do a check like giving "20170217180154Z" and checking that it turned it into "170217180154Z" (and is a V_ASN1_UTCTIME)